store/store.go

```diff
@@ -459,3 +459,7 @@ func (s *Store) GetSize(key []byte) (types.Size, bool, error) {
 	}
 	return blk.Size - types.Size(len(key)), true, nil
 }
+
+func (s *Store) IndexStorageSize() (int64, error) {
```

Comment:
Follow existing naming convention in repo? `GetStorageSize` maybe?

Reply:
Because this is not the total storage size, it is only the size of the index and associated file.  A function `GetStorageSize` should return the index size + the primary storage size.  I will make a comment to clarify.